server/session_cache.go

```diff
@@ -117,7 +117,7 @@ func (s *LocalSessionCache) IsValidSession(userID uuid.UUID, exp int64, tokenId
 		s.RUnlock()
 		return true
 	}
-	if cache.lastInvalidation > 0 && exp-s.tokenExpirySec <= cache.lastInvalidation {
+	if cache.lastInvalidation > 0 && exp-s.tokenExpirySec < cache.lastInvalidation {
```

Comment:
@sesposito Good catch, I've changed the validity check to only deny tokens issued strictly before the full invalidation. Do you see any problems with this?